pkg/operator/controller/controller_dns_configmap.go

```diff
@@ -68,6 +68,11 @@ var corefileTemplate = template.Must(template.New("Corefile").Funcs(template.Fun
     cache {{ $.PositiveTTL }} {
         denial 9984 {{ $.NegativeTTL }}
     }
+    {{- if eq true $.OCPDNSNameResolver}}
+    ocp_dnsnameresolver {
+        namespaces {{range $.DNSNameResolverNamepsaces}}{{.}}{{end}}
```

Comment:
Will we have a space separator? Can you please add a unit test with multiple namespaces?

Reply:
Yes. I have added the space separator and the unit test to check it.